save-core/src/commonMain/kotlin/org/cqfn/save/core/Save.kt

```diff
@@ -103,13 +107,20 @@ class Save(
     }
 
     private fun getReporter(saveProperties: SaveProperties): Reporter {
+        val outFileName = when (saveProperties.reportType!!) {
+            ReportType.PLAIN -> "save.out"
```

Comment:
May be we can create separate variable instead of hardcoded `save.out` in `ReportType`? And here operate just with expansions?

Reply:
Extracted it into a variable for now, we should make it configurable via cli/properties in the future

Reply:
Ok, let's open corresponding issue on this subject

Reply:
#169